docs/user-guide/environment-variables.md

```diff
@@ -80,7 +80,7 @@ These environment variables may or may not be available depending on what plugin
 
 | Name | Description |
 |------|-------|
-| SD_ARTIFACTS_DIR | Location of built/generated files |
+| SD_ARTIFACTS_DIR | Location of built/generated files. _Note: The UI will only display artifacts when builds complete successfully. If a build fails, you can still access the artifacts, but it will need to be done through the store API (e.g.: `https://store.screwdriver.cd/v1/builds/12345/ARTIFACTS/fileName.txt`). You will need to pass in a token along with the request. See the [API docs](./api.md) for more details._ |
```

Comment:
UI does display artifacts that were generated prior to build's end

Reply:
ah okay, I'll update the docs